cmd/clusterawsadm/cmd/alpha/alpha.go

```diff
@@ -17,24 +17,20 @@ limitations under the License.
 package alpha
 
 import (
-	"fmt"
-
 	"github.com/spf13/cobra"
 	"sigs.k8s.io/cluster-api-provider-aws/cmd/clusterawsadm/cmd/alpha/bootstrap"
 	"sigs.k8s.io/cluster-api-provider-aws/cmd/clusterawsadm/cmd/alpha/migrate"
 	"sigs.k8s.io/cluster-api-provider-aws/cmd/clusterawsadm/cmd/flags"
 )
 
 // AlphaCmd is the top-level alpha set of commands
-func AlphaCmd() *cobra.Command { // nolint
+func AlphaCmd() *cobra.Command {
 	newCmd := &cobra.Command{
 		Use:   "alpha",
 		Short: "alpha commands",
 		Long:  `Alpha commands may not be supported in future releases`,
-		Run: func(cmd *cobra.Command, args []string) {
-			if err := cmd.Help(); err != nil {
-				_ = fmt.Errorf("error printing Help")
-			}
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
This was not related to a linter but we were also swallowing this error, it's easy enough to change it to `RunE` and let cobra handle it.